src/js/plugin/PluginDispatcherProxy.js

```diff
@@ -0,0 +1,36 @@
+import PluginDispatcher from "./external/PluginDispatcher";
+
+export default class PluginDispatcherProxy {
```

Comment:
Yeah, but that would mean that we're exposing the `PluginDispatcher` to the plugin. At the moment it's nearly impossible (I'm not aware if the latest plugin sdk removal might have introduced a flaw) for a plugin to dispatch an event to the system without it's plugin id. Thus we can rely on the `pluginId` property and use it in event conditions.